app/blueprints/configuration_blueprint.rb

```diff
@@ -0,0 +1,17 @@
+class ConfigurationBlueprint < Blueprinter::Base
+  association :categories, blueprint: CategoryBlueprint, view: :with_subcategories do
+    Category.visible.roots.includes(:categories)
+  end
+
+  association :contact_methods, blueprint: ContactMethodBlueprint do
+    ContactMethod.enabled
```

Comment:
I'm starting to want the `enabled` to be renamed to `visible` or `display_to_public` or vice versa so they're all the same. Display to public is clearer to me the more I go in and out of the codebase, but, not sure where you're at in this rn? (I know it doesn't/ shouldn't affect this PR)

Reply:
I don't have a strong preference on any of these options and agree consolidating on one would be good. `display_to_public` sounds like a verb to me instead of an adjective, so something like `publicly_visible`? But really don't mind either way.

Reply:
ah. yup. like `publicly_visible`
i think we should do it in a separate PR tho, do you?

Reply:
Agreed.